### ISSUE TITLE
First Lac panda training results

### ISSUE THREAD

**Issue by rickstaa** [rickstaa/stable-learning-control#171]
I trained 3 seeds on the `PandaReach-v1` environment of the https://github.com/rickstaa/ros-gazebo-gym package. After the new initial position fix https://github.com/rickstaa/ros-gazebo-gym/commit/021391c43a52e156c955d3bf826497859fdcfac5, was applied, trained behaviour is a lot smoother (see gif below). On a first look, I'm happy with the smoothness of the control. The agent started with very high-frequency control. There seems to be some training going on, but the agent still has problems reaching the point.

![image](https://user-images.githubusercontent.com/17570430/153414638-3c9a8316-6413-4351-bd40-87b65b9abdd4.png)

![trail](https://user-images.githubusercontent.com/17570430/153414872-c2ce6a8a-a690-4579-a84f-b0a8dbf95d62.gif)

## Possible causes

- [ ] Hyperparameters are not tuned correctly.
    - [ ] Network to simple for such a complex task? 
    - [ ]  Learning rate decrease is too high for long training 300 epochs.
- [ ] We need to use a warm start to control the first trails ourselves.
- [ ] The collision penalty is activated when it should not be activated.
- [ ] The initial position is not randomized enough.
